com.github.vikdevelop.photopea_app.yml

```diff
@@ -0,0 +1,28 @@
+app-id: com.github.vikdevelop.photopea_app
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: photopea_app
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-download
+modules:
+  - shared-modules/dbus-glib/dbus-glib-0.110.json
+  - name: photopea_app
+    buildsystem: simple
+    build-commands:
+    - bash build.sh
+    - install -Dm755 -t /app/share/applications com.github.vikdevelop.photopea_app.desktop
+    - install -Dm644 -t /app/share/icons/hicolor/128x128/apps/ data/icons/com.github.vikdevelop.photopea_app.png
+    - install -Dm644 -t /app/share/metainfo com.github.vikdevelop.photopea_app.metainfo.xml
+   sources:
```

Comment:
still the wrong identation. should be at the same level as `buildsystem` or `build-commands`